### PR TITLE
Tp 54 creating users 02

### PR DESCRIPTION
**Pull request by AliAziz12**

### CODE REVIEW

File: src/main/java/com/project/pufferfish/screen/InvadersScreen.java [hunk 1]
```diff
@@ -34,6 +36,14 @@
     int relX = 0;
     int relY = 0;
 
+    
+    //Player variables
+    public int playerWidth=15, playerHeight=15;
+    Player p = new  Player (textureWidth/2-playerWidth/2,textureHeight-(2*playerHeight));
```
Comment (tharushan-5) [inline src/main/java/com/project/pufferfish/screen/InvadersScreen.java]: Rename 'p' variable to something more defined, like "userSpaceShip" or "invaderPlayer". This way others can know what 'p' is in the class

File: src/main/java/com/project/pufferfish/screen/Player.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	int speed =5;
+	boolean isVisible;
+	public boolean movesRight; 
+	public boolean movesLeft; 
+	public boolean movesUp;
+	
+    public static void main(String[] args)
+    {
+       //	System.out.print("here");
+    }
```
Comment (tharushan-5) [inline src/main/java/com/project/pufferfish/screen/Player.java]: Remove

**Review by tharushan-5 — CHANGES_REQUESTED** [review]
Local testing works as intended. I suggested some small code changes, if you can make the changes I can approve and merge.

File: src/main/java/com/project/pufferfish/screen/InvadersScreen.java (hunk 1 above)
Comment (tharushan-5) [reply src/main/java/com/project/pufferfish/screen/InvadersScreen.java]: Of course, if you have a better variable name, it'd be good to have it here

Comment (AliAziz12) [reply src/main/java/com/project/pufferfish/screen/InvadersScreen.java]: renamed p to tank

File: src/main/java/com/project/pufferfish/screen/Player.java (hunk 2 above)
Comment (AliAziz12) [reply src/main/java/com/project/pufferfish/screen/Player.java]: Done

**Review by tharushan-5 — APPROVED** [review]
Changes look fine. Player class working as intended. Mergin to main.

**Pull request merged**